lib/netsuite/records/base_ref_list.rb

```diff
@@ -2,31 +2,15 @@
 
 module NetSuite
   module Records
-    class BaseRefList
-      include Support::Fields
+    class BaseRefList < Support::Sublist
```

Comment:
@davidlaprade this class is a bit different than the others. It has the `get_select_value` action which can be called on this class directly. Could you try testing this against a live NS instance to see if it functions well? There's an example in the readme.

Otherwise, this PR looks great!!